express_relay/sdk/python/express_relay/searcher/examples/simple_searcher_svm.py

```diff
@@ -50,6 +60,27 @@ def __init__(
         self.limo_client = LimoClient(
             self.rpc_client, global_config=Pubkey.from_string(limo_global_config)
         )
+        self.fill_rate = fill_rate
+        self.express_relay_metadata = None
+
+    async def opportunity_callback(self, opp: Opportunity):
```

Comment:
Maybe the opp here is a OpportunitySvm type?

Reply:
this is the general callback and should accept a general opportunity type. I know it's suboptimal, but I guess the alternative is to have separate callbacks for svm and evm which is ugly in another way 😅